app/main/views/drafts.py

```diff
@@ -0,0 +1,154 @@
+from flask import jsonify, abort
+
+from sqlalchemy.exc import IntegrityError
+
+from .. import main
+from ... import db
+from ...validation import is_valid_service_id_or_400
+from ...models import Service, DraftService, ArchivedService
+from ...service_utils import validate_and_return_updater_request, \
+    update_and_validate_service, validate_and_return_service_request, \
+    index_service
+
+
+@main.route('/services/<string:service_id>/draft', methods=['PUT'])
+def create_draft_service(service_id):
+    """
+    Create a draft service from an existing service
+    :param service_id:
+    :return:
+    """
+    is_valid_service_id_or_400(service_id)
+    updater_json = validate_and_return_updater_request()
+
+    service = Service.query.filter(
+        Service.service_id == service_id
+    ).first_or_404()
+
+    draft = DraftService.from_service(service)
+
+    db.session.add(draft)
+
+    try:
+        db.session.commit()
+        return jsonify(services=draft.serialize()), 201
+    except IntegrityError as e:
+        db.session.rollback()
+        abort(400, "Database Error: {0}".format(e))
+
+
+@main.route('/services/<string:service_id>/draft', methods=['POST'])
+def edit_draft_service(service_id):
+    """
+    Edit a draft service
+    :param service_id:
+    :return:
+    """
+
+    is_valid_service_id_or_400(service_id)
+
+    updater_json = validate_and_return_updater_request()
+    update_json = validate_and_return_service_request(service_id)
+
+    service = DraftService.query.filter(
```

Comment:
Might be nice to call this `draft` rather than `service`, as in the PUT endpoint.